Examples/VoiceMemos/VoiceMemos/VoiceMemos.swift

```diff
@@ -28,6 +28,7 @@ struct VoiceMemo: Equatable {
 enum VoiceMemoAction: Equatable {
   case audioPlayerClient(Result<AudioPlayerClient.Action, AudioPlayerClient.Failure>)
   case playButtonTapped
+  case remove
```

Comment:
I decided to add a new action to the domain of the voice memo row so that it is responsible for cleaning up its effects when it is removed.